### PR TITLE
Biome analyzer

### PR DESCRIPTION
**Pull request by Canop**
For use with https://biomejs.dev/

Example config:

```
[jobs.biome-libs]
env.RAYON_NUM_THREADS = "1" # for constant ordering of items
command = [
    "npx", "@biomejs/biome", "lint",
    "--colors", "force",
    "./libs",
    "--skip", "complexity/useArrowFunction",
    "--skip", "style/useTemplate",
]
need_stdout = true
analyzer = "biome"
watch = ["libs"]
```

Summarized:

image

Not summarized:

image

The env thing gives a reproducible order of items.

### CODE REVIEW

File: src/analysis/biome/mod.rs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                    code,
+                    tag,
+                });
+            }
+        }
+    }
+    None
+}
+
+/// Build a report from the output of eslint
```
Comment (tyilo) [inline src/analysis/biome/mod.rs:89]: Mention biome instead?

Comment (Canop) [reply src/analysis/biome/mod.rs]: done, thanks

**Comment by tyilo** [comment]
I just tested it on a project and it seems to work for me :)

**Comment by Canop** [comment]
Do you see anything missing ?

And btw, do you have an idea how to get a reproducible order of items ?

**Pull request merged**